src/main/resources/org/sonatype/nexus/ci/iq/Messages.properties

```diff
@@ -31,7 +31,7 @@ IqPolicyEvaluation.JobSpecificCredentials=Use job specific credentials
 
 IqPolicyEvaluation.NoIqServersConfigured=No IQ Server configured.
 
-IqPolicyEvaluation.UnableToCommunicate=Unable to communicate with IQ Server: {0}
+IqPolicyEvaluation.Failed=IQ Server request failed for the following reason: {0} cause: {1}
```

Comment:
The actual messages in the tests show that there needs to be some kind of separation between the first exception message and the "cause" + second exception message. I would use:
IQ Server request failed for the following reason: {0} (cause: {1})

Reply:
> IQ Server request failed for the following reason: {0} (cause: {1})

I do think that looks a lot cleaner now that you mention it.